servicer_test.go

```diff
@@ -85,8 +85,8 @@ func TestService_Get(t *testing.T) {
 			header := http.Header{}
 			header.Set(
 				"location",
-				fmt.Sprintf("%s://%s.%s.%s",
-					srv.config.Protocol, name, location, srv.config.Host),
+				fmt.Sprintf("%s://%s.%s/%s",
+					srv.config.Protocol, location , srv.config.Host , name),
```

Comment:
Please make sure the code has been formatted, use `make build` before submitting PR.

Reply:
sorry,remember next time

Reply:
We will introduce a check in CI to avoid this problem: https://github.com/aos-dev/go-storage/issues/563